client/patch-react.js

```diff
@@ -41,12 +41,11 @@ export default (handleError = () => {}) => {
             prototypeWithRender.render = wrapRender(prototypeWithRender.render)
           } else {
             dynamicWrapper = withWrapRenderAlways
```

Comment:
Now it's not wrapping for most of the React component classes. That's why the issue is gone.